manifest.md

```diff
@@ -22,7 +22,9 @@ A client will distinguish a manifest list from an image manifest based on the Co
 - **`mediaType`** *string*
 
   This REQUIRED property contains the MIME type of the manifest list.
-  For this version of the specification, this MUST be set to `application/vnd.oci.image.manifest.list.v1+json`.
+  For this version of the specification, this MUST be set to
```

Comment:
ok.. interesting style, updated to adhere to it. Out of curiosity is there a reason to not break sentences in a .md file?